Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1530,6 +1536,15 @@ public extension Purchases {
         set { purchasesOrchestrator.allowSharingAppStoreAccount = newValue }
     }
 
+    /**
+     * Deprecated. Where responsibility for completing purchase transactions lies.
+     */
+    @available(*, deprecated, message: "Use ``purchasesAreCompletedBy`` instead.")
+    @objc var finishTransactions: Bool {
```

Comment:
Can we move this to the `Purchases` extension in `Deprecations.swift`?

Reply:
Good catch but same issue as discussed in https://github.com/RevenueCat/purchases-ios/pull/3956/files#r1638537384